<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NiceYouGet</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="linkLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>目标链接</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linkLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="cookieTextEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>若目标资源需要登录才能访问，可在此进行cookie填充</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>操作区</string>
    </property>
    <widget class="QLabel" name="cookieLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cookie</string>
     </property>
    </widget>
    <widget class="QPushButton" name="parsePushButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>解析</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>linkLabel</zorder>
   <zorder>linkLineEdit</zorder>
   <zorder>cookieTextEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="actionMinsize"/>
    <addaction name="separator"/>
    <addaction name="actiondownloadHistory"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>关于软件</string>
   </property>
   <property name="statusTip">
    <string>1.0.0</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>首选项</string>
   </property>
   <property name="toolTip">
    <string>首选项</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="actionMinsize">
   <property name="text">
    <string>最小化</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actiondownloadHistory">
   <property name="text">
    <string>下载记录</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
